spec/bundle/commands/install_command_spec.rb

```diff
@@ -14,6 +14,16 @@
     end
   end
 
+  context "when an installer raises an error" do
+    it "rescues the error" do
+      allow(ARGV).to receive(:value).and_return(nil)
+      allow(Bundle::MacAppStoreInstaller).to receive(:install).and_throw(RuntimeError)
```

Comment:
This doesn't seem to describe the issue that's mentioned in https://github.com/Homebrew/homebrew-bundle/issues/456. `mas` will only throw Ruby exceptions on invalid format in the `Brewfile`; I'm not sure I understand how catching and reporting this helps?